README.ci.md

```diff
@@ -1,14 +1,14 @@
 **WARNING:** This document is a work in progress and intended as a RFC.
-If you are not a Coq Developer, don't follow this instructions yet.
+If you are not a Coq Developer, don't follow these instructions yet.
 
 Introduction
 ============
 
-The Coq Travis CI infrastructure is meant to provide lightweight
-automatics testing of pull requests.
-If you are on GitLab, their integrated CI is also set up.
+As of 2017, lightweight tests are automatically run against all pull requests
```

Comment:
I noted the year because some people reading README.ci.md might not have a record of the file's revision history. Such readers may benefit from knowing the rough time period from which the file became existent and relevant (e.g. in order to form a view of how fresh or stale its claims are), especially given that it describes itself as a WiP and RFC.